index.js

```diff
@@ -93,6 +93,13 @@ function fastifyRedis (fastify, options, next) {
       onEnd(err)
       return
     }
+    if (err.code === 'SELF_SIGNED_CERT_IN_CHAIN') {
+      // This error is not recoverable because ioredis will never be able to connect to the server unless the user changes the TLS options.
+      // Provide a more helpful error message to the user so they understand how to fix the issue instead of the plugin timing out.
+      const error = new Error("Self-signed certificate detected in the certificate chain. Consider setting the 'tls.rejectUnauthorized' option to 'false' or providing a valid certificate using the 'tls.ca' option.", { cause: err })
```

Comment:
I think it is a bad idea to suggest `tls.rejectedUnauthorized`. At the very least, it should not be the _first_ suggestion. I think there should be an article in the "Learn" section of nodejs.org detailing how to solve this error. This error could then link to it.

Reply:
Removed the custom message in [this commit](https://github.com/fastify/fastify-redis/pull/236/commits/5bed1632766c242d70816b5e28bbe453adb095ac)